README.md

```diff
@@ -53,9 +59,22 @@ After Bower is installed, you can install all of the edX UX Pattern Library depe
 bower install
 ```
 
-NOTE: Bower packages are installed to the ``_src/components`` directory. These files should not be edited or removed outside of the Bower workflow.
+**NOTE**: Bower files should not be edited or removed outside of the Bower workflow.
+
+- - -
+
+### Step 3: Install NodeJS
```

Comment:
Amazeballs catch! Yes, it does. I've switched the order of instructions in https://github.com/edx/ux-pattern-library/commit/72c2a49a037c4d863862af0dc159b0cfc6e86bdd. Thanks!